addons/account/tests/test_chart_template.py

```diff
@@ -318,6 +318,45 @@ def local_get_data(self, template_code):
             {'name': 'Tax 4'},
         ])
 
+    def test_remove_fiscal_position_try_loading_force_create_false(self):
+        """Test that removing a fiscal position and calling try_loading with force_create=False does not recreate it."""
+
+        # Ensure the fiscal position exists
+        fiscal_position = self.env['account.fiscal.position'].search([
+            ('company_id', '=', self.company.id),
+            ('name', '=', 'Fiscal Position')
+        ])
+        self.assertTrue(fiscal_position, "Fiscal Position should exist before deletion")
+
+        # Remove related fiscal position tax mappings
+        fiscal_position_taxes = self.env['account.fiscal.position.tax'].search([
+            ('position_id', '=', fiscal_position.id)
+        ])
+        fiscal_position_taxes.unlink()
```

Comment:
Why do you need to do that?

Reply:
I thought that we should remove the taxs first then remove the fiscal position but it seams that it is not related and that we could remove directly the fiscal position :+1:   :)